### PR TITLE
OBSDOCS-1388 - 5.9.8 4.13-4.16

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.13, 4.14, 4.15, 4.16

Issue: [OBSDOCS-1388](https://issues.redhat.com//browse/OBSDOCS-1388)

Link to docs preview: https://83940--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/logging_release_notes/logging-5-9-release-notes.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kabirbhartiRH** [comment]
Please add https://issues.redhat.com/browse/LOG-6214

File: modules/logging-release-notes-5-9-8.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+(link:https://issues.redhat.com/browse/LOG-6181[LOG-6181])
+
+* Before this update, the LokiStack ruler component view did not initialize properly, causing an invalid field error when the ruler component was disabled. This update ensures that the component view initializes with an empty value, resolving the issue.
+(link:https://issues.redhat.com/browse/LOG-6183[LOG-6183])
+
+* Before this update, an LF character in the `vector.toml` file under the ES authentication configuration caused the collector pods to crash. This update removes the newline characters from the username and password fields, resolving the issue.
+(link:https://issues.redhat.com/browse/LOG-6206[LOG-6206])
+
+[id="openshift-logging-5-9-8-CVEs_{context}"]
+== CVEs
```
Comment (kabirbhartiRH) [inline modules/logging-release-notes-5-9-8.adoc:24]: The Errata contains more CVE's

**Review by JoaoBraveCoding — COMMENTED** [review]
Apart from the missing CVE's lgtm

File: modules/logging-release-notes-5-9-8.adoc (hunk 1 above)
Comment (libander) [reply modules/logging-release-notes-5-9-8.adoc:24]: Not from what I see? 

image

**Review by jcantrill — COMMENTED** [review]
lgtm

**Comment by libander** [comment]
Replacing with https://github.com/openshift/openshift-docs/pull/84071.

**Pull request closed without merging**